File: python/2017/4/solution2.py
with open("../../../data/2017/4/data.txt", "rb") as f:
    lines = [line.replace('\n','').split() for line in f.readlines()]

count = 0
for line in lines:
    invalid = False
    for i in range(len(line)):
        for j in range(i+1, len(line)):
            if set(list(line[i])) == set(list(line[j])):
                invalid = True
    if not invalid:
        count += 1

print "Solution is %d" % count

File: julia/2017/4/solution2.jl
f = open("../../../data/2017/4/data.txt")
lines = readlines(f)
close(f)

count = 0
for line in lines
    invalid = false
    line = split(line, " ")
    for i = 1:length(line)
        for j = i+1:length(line)
            if Set(collect(line[i])) == Set(collect(line[j]))
                invalid = true
            end
        end
    end
    if !invalid
        count += 1
    end
end

println("Solution is $count")
